app/src/test/java/waiter/EchoServerTest.java

```diff
@@ -0,0 +1,44 @@
+package waiter;
+
+import waiter.Communicator.mock.CommunicatorMock;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.io.PrintStream;
+import java.io.ByteArrayOutputStream;
+
+import java.io.IOException;
+import java.net.SocketException;
+
+public class EchoServerTest {
```

Comment:
```suggestion
class EchoServerTest {
```

Non-blocking: I _think_ with JUnit5 you can depend on the default package visibility for test files.

Reply:
Addressed in [this commit](https://github.com/jonmanahan/waiter/pull/4/commits/1c0ece08eec1ec5aa9d82701db2be444022898f6), thank you!